### PR TITLE
Add medical insurance to patients

### PR DESCRIPTION
**Pull request by DaveFIND**

### CODE REVIEW

File: cdx_core/app/controllers/patients_controller.rb [hunk 1]
```diff
@@ -89,7 +89,7 @@ def destroy
 
   def patient_params
     params.require(:patient).permit(
-      :name, :entity_id, :gender, :nickname, :social_security_code, :birth_date_on, :address, :email, :phone, :city, :state, :zip_code,
+      :name, :entity_id, :gender, :nickname, :medical_insurance_num, :social_security_code, :birth_date_on, :address, :email, :phone, :city, :state, :zip_code,
```
Comment (tripledub) [inline cdx_core/app/controllers/patients_controller.rb]: `:medical_insurance_num` doesn't belong here

Comment (DaveFIND) [inline cdx_core/app/controllers/patients_controller.rb]: why?  It's a permitted parameter for the form.

Comment (tripledub) [inline cdx_core/app/controllers/patients_controller.rb]: Yeah but only for Vietnam y/n?

Comment (DaveFIND) [inline cdx_core/app/controllers/patients_controller.rb]: No.  It's a core field too. (Ticket 2196)

Comment (tripledub) [inline cdx_core/app/controllers/patients_controller.rb]: OK, I stand corrected

Comment (DaveFIND) [inline cdx_core/app/controllers/patients_controller.rb]: Nil problemski.

**Comment by jorgegorka** [comment]
I have moved the Medical insurance migration to cdx_core.

Please, remember to keep all core functionality inside cdx_core.

Thanks

**Pull request merged**